src/com/opc/opccomlayer.cpp

```diff
@@ -36,11 +36,19 @@ COpcComLayer::COpcComLayer(CComLayer* paUpperLayer, CBaseCommFB* paComFB) :
 COpcComLayer::~COpcComLayer(){
   DEVLOG_DEBUG("delete OpcComLayer: [0x%lX][%s]\n", this, mOpcGroupName.c_str());
   for(auto &it :mFBInputVars){
-    delete it;
+    if(nullptr != it){
```

Comment:
according to the specification delete shall correctly handle nullptr. Therefore the nullptr check is not needed.